packages/cf-deploy-config-inquirer/test/conditions.test.ts

```diff
@@ -0,0 +1,35 @@
+import { showManagedAppRouterQuestion, showOverwriteQuestion } from '../src/prompts/conditions';
+
+describe('showManagedAppRouterQuestion', () => {
```

Comment:
Please test by calling the when clause of the prompt and remove the `showABC` patterns, this was legacy from deploy config.

Reply:
Gone stale .. since we now display Managed App Router & Overwrite prompts only if enabled.